test/integration/proxy_proto_integration_test.cc

```diff
@@ -137,7 +137,7 @@ TEST_P(ProxyProtoIntegrationTest, AccessLog) {
 
   ASSERT_EQ(2, tokens.size());
   EXPECT_EQ(tokens[0], Network::Test::getLoopbackAddressString(GetParam()));
-  EXPECT_EQ(tokens[1], "1.2.3.4:12345\n");
+  EXPECT_EQ(tokens[1], fmt::format("1.2.3.4:12345{}", TestEnvironment::newLine));
```

Comment:
Trying to wrap my head around why proxy proto would behave different for Windows/Linux? Presumably new lines here are network wire features, not files.

Reply:
I haven't delved much into it, initially I though it came from the way we save the access log file but I could be wrong. I will investigate a bit more and get back at you.

Reply:
Carriage control/line endings seem really weird in this context, I'd expect all whitespace to fall off once tokens are split.

Reply:
by default `split` does not trim whitespace. But that would be an alternative way to get the test working.

Reply:
I think that would be more robust in light of different type of line endings.